recipe/build.sh

```diff
@@ -5,21 +5,25 @@
 if [[ "$cuda_compiler_version" == "None" ]]; then
   export FORCE_CUDA=0
 else
-  export TORCH_CUDA_ARCH_LIST="3.5;5.0+PTX"
   if [[ ${cuda_compiler_version} == 9.0* ]]; then
-      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;7.0"
+      export TORCH_CUDA_ARCH_LIST="3.5;5.0;6.0;7.0+PTX"
   elif [[ ${cuda_compiler_version} == 9.2* ]]; then
-      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0"
+      export TORCH_CUDA_ARCH_LIST="3.5;5.0;6.0;6.1;7.0+PTX"
   elif [[ ${cuda_compiler_version} == 10.* ]]; then
-      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5"
+      export TORCH_CUDA_ARCH_LIST="3.5;5.0;6.0;6.1;7.0;7.5+PTX"
   elif [[ ${cuda_compiler_version} == 11.0* ]]; then
-      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0"
+      export TORCH_CUDA_ARCH_LIST="3.5;5.0;6.0;6.1;7.0;7.5;8.0+PTX"
   elif [[ ${cuda_compiler_version} == 11.1* ]]; then
-      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0;8.6"
+      export TORCH_CUDA_ARCH_LIST="3.5;5.0;6.0;6.1;7.0;7.5;8.0;8.6+PTX"
```

Comment:
IMO we should just delete everything pre-11.2, which is the lowest supported version in conda-forge for now.

Reply:
Sounds good! less code = less bugs, since this is just for building on conda-forge, it makes sense to keep this list up to date with what we actually build